package.json

```diff
@@ -11,6 +11,7 @@
     "mongoose": "6.x"
   },
   "devDependencies": {
+    "azure-functions-core-tools": "^4.0.3971",
```

Comment:
This change looks accidental, can you remove this please?

Trying to avoid adding azure because azure is huge and makes `npm install` take forever

Reply:
I can remove it but I put in dev dependencies because I need it when I develop on this repo. I'll just make a note in the read me for myself.